bundle/config/workspace.go

```diff
@@ -91,6 +88,13 @@ func (w *Workspace) Client() (*databricks.WorkspaceClient, error) {
 		}
 	}
 
+	if w.Profile != "" {
```

Comment:
This should also check if the host is set, because we assume so in the validate function.